src/packages/client/src/__tests__/integration/errors/missing-binary-native/test.ts

```diff
@@ -35,31 +35,31 @@ test('missing-binary-native', async () => {
+                                    You incorrectly pinned it to TEST_PLATFORM
 
-            This probably happens, because you built Prisma Client on a different platform.
-            (Prisma Client looked in "/client/src/__tests__/integration/errors/missing-binary-native/node_modules/@prisma/client/runtime/libquery_engine_napi-TEST_PLATFORM.so.node")
+                                    This probably happens, because you built Prisma Client on a different platform.
+                                    (Prisma Client looked in "/client/src/__tests__/integration/errors/missing-binary-native/node_modules/@prisma/client/runtime/libquery_engine_napi-TEST_PLATFORM.so.node")
 
-            Searched Locations:
+                                    Searched Locations:
 
-              /client/src/__tests__/integration/errors/missing-binary-native/node_modules/.prisma/client
-              /client/src/__tests__/integration/errors/missing-binary-native/node_modules/@prisma/client/runtime
-              /client/src/__tests__/integration/errors/missing-binary-native/node_modules/@prisma/client
-              /client/src/__tests__/integration/errors/missing-binary-native/node_modules/.prisma/client
-              /client/src/__tests__/integration/errors/missing-binary-native
-              /tmp/prisma-engines
-              /client/src/__tests__/integration/errors/missing-binary-native/node_modules/.prisma/client
+                                      /client/src/__tests__/integration/errors/missing-binary-native/node_modules/.prisma/client
+                                      /client/src/__tests__/integration/errors/missing-binary-native/node_modules/@prisma/client/runtime
+                                      /client/src/__tests__/integration/errors/missing-binary-native/node_modules/@prisma/client
+                                      /client/src/__tests__/integration/errors/missing-binary-native/node_modules/.prisma/client
+                                      /client/src/__tests__/integration/errors/missing-binary-native
+                                      /tmp/prisma-engines
+                                      /client/src/__tests__/integration/errors/missing-binary-native/node_modules/.prisma/client
 
-            You already added the platform "native" to the "generator" block
-            in the "schema.prisma" file as described in https://pris.ly/d/client-generator,
-            but something went wrong. That's suboptimal.
+                                    You already added the platform "native" to the "generator" block
+                                    in the "schema.prisma" file as described in https://pris.ly/d/client-generator,
+                                    but something went wrong. That's suboptimal.
 
-            Please create an issue at TEST_GITHUB_LINK
-          `)
+                                    Please create an issue at TEST_GITHUB_LINK
+                              `)
```

Comment:
What is going on here? That this changed is weird.

Reply:
inline snapshots are weird sometimes 🤷‍♂️